### PR TITLE
Milestone 1: Used Linters HTML & CSS in a project

### PR DESCRIPTION
**Pull request by mdg0410**
## Linters HTML & CSS Milestone 1:   Used Linters HTML & CSS in a project

In this milestone:

- Created folder `.github/workflows` and in this folder created `linters.yml`
- Added the linters HTML & CSS `Webhint` & `Stylelint`
- Created `.gitignore` file and typing `node_modules` and `test.md` for ignore this archives
- Created `index.html` file with messange `Hello world!`
- Created `styles.css` file and was style `body` and `h1` tag
- Confirm that the file `test.md` not submit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,152 @@
-# Hello_World
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="https://github.com/microverseinc/readme-template/raw/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
Comment (RaoAkif) [inline README.md]: [OPTIONAL] Remove the Microverse logo from the README file. You have to customize it for your project.

**Review by RaoAkif — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @mdg0410, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (mdg0410) [reply README.md]: Hello @RaoAkif 
I have modified it and proceed to make the merge.

Comment (RaoAkif) [reply README.md]: Perfect 👌🏻

**Pull request merged**